### PR TITLE
feat: better gracefully shutdown

### PR DESCRIPTION
**Pull request by Timeless0911**
## Summary

Actively exit the process when gracefully shutdown.

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: packages/cli/src/index.ts
Comment (chenjiahan) [inline packages/cli/src/index.ts:1]: The PR title can be `feat` instead of `chore`.

Comment (Timeless0911) [reply packages/cli/src/index.ts:1]: done

**Review by chenjiahan — APPROVED** [review]

File: packages/cli/src/index.ts [hunk 1]
```diff
@@ -100,9 +100,14 @@ cli
       await startDevServer();
 
       const exitProcess = async () => {
```
Comment (9aoy) [inline packages/cli/src/index.ts:102]: shall we add a timeout to force exit after 5 seconds?

Comment (chenjiahan) [reply packages/cli/src/index.ts:102]: 5 seconds too too long

Comment (Timeless0911) [reply packages/cli/src/index.ts:102]: Welcome your pr 😀

**Pull request merged**